<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPocket</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/search.png</normaloff>images/search.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="list1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>671</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>721</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>541</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="xButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/x.png</normaloff>images/x.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="list2">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>90</y>
      <width>161</width>
      <height>421</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>50</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Tags</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
